cluster-autoscaler/metrics/metrics.go

```diff
@@ -422,6 +443,16 @@ func UpdateMemoryLimitsBytes(minMemoryCount int64, maxMemoryCount int64) {
 	memoryLimitsBytes.WithLabelValues("maximum").Set(float64(maxMemoryCount))
 }
 
+// UpdateNodeGroupMin records the node group minimum allowed number of nodes
+func UpdateNodeGroupMin(nodeGroup string, minNodes int) {
+	nodesGroupMinNodes.WithLabelValues(nodeGroup).Set(float64(minNodes))
```

Comment:
What happens if we set value of unregistered metric? Is this supported? Or are we relying on some undocumented implementation detail here?

Reply:
Setting the value is independent from registering the metrics for collection. In the standard Prometheus client, the value will be set, and can be read to what it was set to, but it will not be exposed to collection unless it is registered. In the current wrapper implementation used, internal the Prometheus gauge is not created unless it is registered, and hence the set operation is a no-op.